src/main/java/nextstep/qna/domain/Question.java

```diff
@@ -65,28 +58,39 @@ public NsUser getWriter() {
 
     public void addAnswer(Answer answer) {
         answer.toQuestion(this);
+        //List<Answer> newAnswers = new ArrayList<>();
+        //newAnswers.add(answer);
+        if(answers == null) {
+            answers = new Answers(new ArrayList<>());
```

Comment:
```suggestion
            answers = new Answers(Collections.emptyList());
```

빈 리스트라면 제공하고 있는 `emptyList`를 사용하는 것을 권장드려요
왜 그럴까요?

Reply:
두 방법의 핵심적인 차이점는 불변성이군요!

Reply:
emptyList는 이미 생성되어있는 빈 리스트를 반환해요
이런 점 때문에 새로운 객체를 생성하지 않아도 된다는 장점이 있습니다.

<img width="659" alt="image" src="https://github.com/user-attachments/assets/5a79bae3-8349-491a-865a-3d2826c256a4">

Reply:
감사합니다!